spec/features/author_show_spec.rb

```diff
@@ -75,4 +75,37 @@
       expect(page).to have_link("Steve", href: author_path(author_3))
     end
   end
+
+  it 'shows the top review information for each book on the page' do
+    author = Author.create(name: 'bob')
+    book = author.books.create(thumbnail: 'steve.jpg', title: 'where the wild things are', pages: 40, year_published: 1987)
+    book.reviews.create(rating: 3, title: "whatever", description: "whahhednd vijnvsihb", username: "harbi")
+    book.reviews.create(rating: 2, title: "is horrible", description: "whahhednd vijnvsihb", username: "stub")
+    book.reviews.create(rating: 1, title: "super bad", description: "whahhednd vijnvsihb", username: "rude")
+    book.reviews.create(rating: 4, title: "haha", description: "whahhednd vijnvsihb", username: "rob")
+    book.reviews.create(rating: 5, title: "meh", description: "whahhednd vijnvsihb", username: "bob")
+
+    book_2 = author.books.create(thumbnail: 'steve.jpg', title: 'Whatever the whatever', pages: 40, year_published: 1987)
+    book_2.reviews.create(rating: 3, title: "hammy", description: "whahhednd vijnvsihb", username: "harbi")
+
+    visit author_path(author)
+
+    within "#book-#{book.id}" do
+      expect(page).to have_content("Top Review:")
+      expect(page).to have_content("Rating: 5 out of 5")
+      expect(page).to have_content("Title: meh")
+      expect(page).to have_content("User: bob")
+      expect(page).to have_link("bob")
+      click_link 'bob'
```

Comment:
What is this line 99 click_link 'bob' for?

Reply:
That was placed as a quick check for the user show links working. But I added a second book to make sure it was populating correctly and didn't move it. I moved it to sit below the withins and before the updated path spec so it will not break the other tests.

Reply:
Ok, that makes sense now.